viewing_party/utils_05.py

```diff
@@ -0,0 +1,23 @@
+from .utils_03 import *
+
+def get_new_rec_by_genre(user_data):
+    genres = set([movie['genre']
+        for movie in user_data['watched']
+    ])
+
+    movies_user_not_watched = [movie['title'] for movie in get_friends_unique_watched(user_data)]
```

Comment:
Great use of helper functions, sets, and list comprehensions to write clear, concise code.